viewing_party/party.py

```diff
@@ -1,23 +1,229 @@
-# ------------- WAVE 1 --------------------
+# Assumption: all strings from input are in the expected case
+# Assumption: the movie 'rating' is an overall public rating, not personal 
+# (will be same for user/friend)
 
+# ------------- WAVE 1 --------------------
+#TC: O(1) ; SC: O(1) 
 def create_movie(title, genre, rating):
-    pass
+    """
+    create a movie dict with three fields provided;
+    Parameters: title(str), genere(str), and rating(int);
+    Return dict
+    """
+    if not title or not genre or not rating:
+        return None
+    return {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+        }

```

Comment:
Tiniest little nitpick here, but traditionally, we'll keep the closing bracket for a dictionary in line with the first line it gets introduced. 
```suggestion
    return {
        "title": title,
        "genre": genre,
        "rating": rating
    }

```